wechatpay/src/main/java/com/adyen/checkout/wechatpay/WeChatPayUtils.kt

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright (c) 2019 Adyen N.V.
+ *
+ * This file is open source and available under the MIT license. See the LICENSE file for more info.
+ *
+ * Created by caiof on 17/9/2019.
+ */
+package com.adyen.checkout.wechatpay
+
+import android.app.Application
+import android.content.Intent
+import com.adyen.checkout.components.model.payments.response.WeChatPaySdkData
+import com.adyen.checkout.core.exception.CheckoutException
+import com.tencent.mm.opensdk.constants.Build
+import com.tencent.mm.opensdk.modelbase.BaseResp
+import com.tencent.mm.opensdk.modelpay.PayReq
+import com.tencent.mm.opensdk.openapi.WXAPIFactory
+import org.json.JSONException
+import org.json.JSONObject
+
+object WeChatPayUtils {
+
+    private const val RESULT_EXTRA_KEY = "_wxapi_baseresp_errstr"
+    private const val RESULT_CODE = "resultCode"
+
+    fun isResultIntent(intent: Intent?): Boolean {
+        return intent != null && intent.extras != null && intent.extras!!.containsKey(RESULT_EXTRA_KEY)
```

Comment:
Can this be replaced with `intent?.extras?.containsKey(RESULT_EXTRA_KEY) == true`?